db/memtable.h

```diff
@@ -423,8 +423,14 @@ class MemTable {
   // which has been inserted into this memtable.
   std::atomic<uint64_t> min_prep_log_referenced_;
 
+  struct AlignedRWMutex {
+    port::RWMutex rw_mutex;
+    char padding[(CACHE_LINE_SIZE - sizeof(port::RWMutex) % CACHE_LINE_SIZE) %
+                 CACHE_LINE_SIZE];
+  };
+  static_assert(sizeof(AlignedRWMutex) % 64 == 0, "Expected 64-byte aligned");
   // rw locks for inplace updates
-  std::vector<port::RWMutex> locks_;
+  std::vector<AlignedRWMutex> locks_;
```

Comment:
Although almost surely for large number of locks, there is no guarantee single AlignedRWMutex is allocated in cache line boundray, right?